wagtail/admin/templates/wagtailadmin/shared/panel.html

```diff
@@ -19,23 +19,23 @@
 {% fragment as panel_id %}{{ prefix }}-section{% endfragment %}
 {% fragment as heading_id %}{{ prefix }}-heading{% endfragment %}
 {% fragment as content_id %}{{ prefix }}-content{% endfragment %}
-<section class="w-panel {{ classname }}" id="{{ panel_id }}" {% if heading %}aria-labelledby="{{ heading_id }}"{% endif %} data-panel>
+<section class="w-panel {{ classname }}" id="{{ panel_id }}" {% if heading %}aria-labelledby="{{ heading_id }}"{% endif %} w-panel data-controller="w-panel" >
```

Comment:
```suggestion
<section class="w-panel {{ classname }}" id="{{ panel_id }}" {% if heading %}aria-labelledby="{{ heading_id }}"{% endif %} data-controller="w-panel" >
```

We should not need the original data attributes at all once this PR is done.